about/about.css

```diff
@@ -0,0 +1,83 @@
+*{
+    margin:0;
+    padding: 0;
+}
+.background-image{
+    background-size: cover; 
+    height:500px; 
+    padding-top:80px;
+    text-align:left;
+}
+.box {
+    background: url(images/codecomplogo.jpg);
+    padding: 100px;
+    margin: 100px;
+}
+h1 {
+    font-size:70px; 
+    color:hsl(225, 100%, 23%); 
```

Comment:
No hard-coded color values: this should be in the root as a CSS variable

Reply:
Please try to adopt a similar to theme to the other pages. See the profile page as an example.